resolver/resolver_test.go

```diff
@@ -2140,6 +2141,40 @@ func TestLiteral(t *testing.T) {
 	}
 }
 
+func TestValidation(t *testing.T) {
```

Comment:
The validations themselves are implemented in the resolver.go, so I put the test here. Also, it seems invalid proto files are in the validator/testdata directory, so I put the test data files there.